tests/test_subdomains.py

```diff
@@ -672,29 +672,6 @@ class Dummy(SubDomainSet):
         assert z.is_Parallel
 
 
-class TestMultiSubDimension:
```

Comment:
Removed test?

Reply:
There were issues rebuilding `MultiSubDimension` because of how the thickness worked, so it has a bunch of extra gubbins to handle this. This PR removes the thickness as a kwarg and all the extra machinery, so this test isn't really needed anymore